leakcanary-android-core/src/main/java/leakcanary/internal/InternalLeakCanary.kt

```diff
@@ -285,6 +299,14 @@ internal object InternalLeakCanary : (Application) -> Unit, OnObjectRetainedList
     }
   }
 
+  private fun getHeapDumpPreferences(): SharedPreferences {
```

Comment:
could this be a lazy instead? That way the strict mode change only happens once?

Reply:
yes, made the initialization lazy